superset-frontend/src/common/components/Tabs.tsx

```diff
@@ -17,9 +17,9 @@
  * under the License.
  */
 import styled from '@superset-ui/style';
-import { Tabs as BaseTabs } from 'src/common/components';
+import { Tabs as AntdTabs } from 'src/common/components';
```

Comment:
Figured we might as well call these things what they really are, as a convention.